lua/nvim-highlight-colors/utils.lua

```diff
@@ -199,20 +199,37 @@ function M.highlight_with_lsp(active_buffer_id, ns_id, positions, options)
 
 	for _, client in pairs(clients) do
 		if client.server_capabilities.colorProvider then
-			client.request(
-				"textDocument/documentColor",
-				param,
-				function(_, response)
-					M.highlight_lsp_document_color(
-						response,
-						active_buffer_id,
-						ns_id,
-						positions,
-						options
-					)
-				end,
-				active_buffer_id
-			)
+			if vim.fn.has('nvim-0.11') then
```

Comment:
Please use `vim.version`, similar to what was done here: https://github.com/brenoprata10/nvim-highlight-colors/blob/7eeaa05eab26fb840c4fe7bca91710bf37cb13ed/lua/nvim-highlight-colors/utils.lua#L163

Reply:
Thanks, hadn't realized that was a thing, pushed an update.